### ISSUE TITLE
Enable BashOnWindows

### ISSUE THREAD

**Issue by giggio** [rogalmic/vscode-bash-debug#29]
You mention that the extension works with some hacks on Bash On Windows, but I could not get it work. Can you describe how to do it?

**Comment by rogalmic** [comment]
1. Use Windows 10 insider build 15014 or newer
2. Install bash on windows (via add/remove system features)
3. install bashdb (in bash "sudo apt-get install bashdb")
4. install extension in vscode under windows (if it is not there already)

This should work with newest version of extension, but it is highly experimental.

**Comment by giggio** [comment]
I was getting an error, even though I am on latest Windows 10 build.

````
Error: write EPIPE
````

![image](https://cloud.githubusercontent.com/assets/334958/24757291/1801c34a-1adf-11e7-9b09-b187fbd390c3.png)

The problem is that I was changing the launch.json file, on the bashPath, from `sysnative` to `system32`. I realized I shouldn't be doing that, and that `sysnative` is where `bash.exe` is, when running a 32 bit app, like VSCode.

So, without changing anything, it worked. Thanks.

I posted the problem and solution, just in case someone tries to be smart and fails miserably, like me. :)

Cheers
